packages/client/src/orders/types/orderShippingAddressChangeRequests.types.ts

```diff
@@ -0,0 +1,31 @@
+import type { CheckoutAddress } from '../../types';
+import type { CheckoutOrder } from '../../checkout';
+import type { Order } from './order.types';
+
+export type OrderShippingAddressChangeRequests =
+  Array<OrderShippingAddressChangeRequest>;
+
+export type OrderShippingAddressChangeRequest = {
+  id: string;
+  checkoutOrderId: CheckoutOrder['id'];
+  orderId: Order['id'];
+  status: OrderShippingAddressChangeRequestStatus;
```

Comment:
According to the endpoint, this should be a string, so I think the definition of the `OrderShippingAddressChangeRequestStatus` should be with string values instead of number values. Can you confirm please?

Reply:
You are right. Fixed 👍